web/src/components/netflow-record/record-field.tsx

```diff
@@ -185,9 +158,6 @@ export const RecordField: React.FC<{
       case ColumnsId.collectiontime:
       case ColumnsId.starttime:
       case ColumnsId.endtime:
-      case ColumnsId.dnsrequesttime:
-      case ColumnsId.dnsresponsetime:
-        return dateTimeContent(typeof value === 'number' && !isNaN(value) ? new Date(value) : undefined);
```

Comment:
This `dateTimeContent` call must be kept, for all the other date/time fields which were also using it.